app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -422,8 +430,28 @@ export class CreateBranch extends React.Component<
         this.state.startPoint === StartPoint.UpstreamDefaultBranch
           ? this.state.startPoint
           : StartPoint.CurrentBranch
+      return (
+        <div>
+          {this.renderForkLink()}
+          {this.renderOptions(items, selectedValue)}
+        </div>
+      )
+    }
+  }
 
-      return this.renderOptions(items, selectedValue)
+  private renderForkLink = () => {
+    if (isRepositoryWithForkedGitHubRepository(this.props.repository)) {
+      return (
+        <p>
+          Fork behaviour can be changed from {' '}
```

Comment:
This should probably match the American English spelling used in repository settings:

```suggestion
          Fork behavior can be changed from {' '}
```